tango/__main__.py

```diff
@@ -105,6 +108,14 @@ class TangoGlobalSettings(FromParams):
     By default, it is set to ``False``.
     """
 
+    multiprocessing_start_method: str = "spawn"
```

Comment:
Turns out the start method needs to be set as early as possible, and only "spawn" and "forkserver" work with PyTorch DDP on GPU.